DependencyInjection/Compiler/TagSubscriberPass.php

```diff
@@ -25,12 +25,18 @@ class TagSubscriberPass implements CompilerPassInterface
     public function process(ContainerBuilder $container)
     {
         if (true === $container->getParameter('fos_http_cache.compiler_pass.tag_annotations')
-            && !$container->has('sensio_framework_extra.controller.listener')
+            && !$this->hasControllerListener()
         ) {
             throw new \RuntimeException(
                 'Tag support requires SensioFrameworkExtraBundle’s ControllerListener for the annotations. '
                 .'Please install sensio/framework-extra-bundle and add it to your AppKernel.'
             );
         }
     }
+
+    private function hasControllerListener()
+    {
+        return $container->has('sensio_framework_extra.controller.listener') ||
+            $container->has('Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener');
```

Comment:
please use `\\` to make this strictly correct. 

in master we can use the ::class constant, but 1.3 unfortunately still supports php 5.3